tutorial/README.md

```diff
@@ -128,6 +130,19 @@ link by appending an `image_url` query parameter:
 > [HTTP range requests](https://developer.mozilla.org/en-US/docs/Web/HTTP/Range_requests) to support viewing OME-TIFF images.
 > Range requests are allowed by default by `http-server` but may need to be enabled explicitly for your production web server.
 
+#### Viewing an Image via SSH
+
+If you would like to follow this tutorial (i.e downloading the data and running the image pyramid pipeline) on a remote machine into which you would ssh,
+you can use port forwarding to visualize the data that results from this tutorial.
+That is, before starting this tutorial, you would ssh into the remote machine via `ssh -L 12345:localhost:8000 myserver.tosshto.com`.
```

Comment:
Not sure how to suggest changes with inner markdown...

"""
For example, you can follow this tutorial on a remote machine within SHH, linking your local port `12345` to the remote's local `8000`.

```bash
$ ssh -L 12345:localhost:8000 <username>@<serverhost> # setup secure shell, link ports
# ... follow tutorial within secure shell, starting http-server on port 8000
```

Since your local port `12345` is linked to the remote `8000` via SSH, you can now view the remote dataset locally via your `localhost:12345` in Avivator: http://avivator.gehlenborglab.org/?image_url=http://localhost:12345/LuCa-7color_Scan1.ome.tif.
"""

Reply:
Thanks, mcuh clearer!